extensions/css-language-features/client/src/cssMain.ts

```diff
@@ -30,13 +30,30 @@ export function activate(context: ExtensionContext) {
 
 	let documentSelector = ['css', 'scss', 'less'];
 
+	let dataPaths: string[] = workspace.getConfiguration('css').get('experimental.customData', []);
```

Comment:
I suspect this is just for teting purposes, but I assume in the end, these files will not be located in the workspace but part of a extension, correct?

Reply:
They'll, see an example for HTML: https://github.com/octref/web-components-examples

Don't think we can assume everyone wants to publish extensions for enhancing their CSS.